detekt.yml

```diff
@@ -16,6 +16,8 @@ potential-bugs:
 complexity:
   LongParameterList:
     threshold: 8
+  StringLiteralDuplication:
+    active: false
```

Comment:
started hitting this in tests (which feels kind of common).  I generally dont' mind some string duplication in tests if it means i don't have to scroll up to see what constant values are etc... don't hold strongly to this though

Reply:
I'm cool with this